.github/workflows/aws-deploy.yaml

```diff
@@ -12,70 +12,26 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: pre-commit/action@v2.0.0
-  sceptre-sandbox:
-    if: github.event_name == 'push'
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-deploy:
+    name: Deploy CloudFormation templates using sceptre
     runs-on: ubuntu-latest
     needs: pre-commit
+    if: github.ref == 'refs/heads/prod'
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.8
-      - name: Cache python modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install sceptre sceptre-ssm-resolver
+      - name: Checkout repository
+        uses: actions/checkout@v2
       - name: Assume AWS Role
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
-          aws-secret-access-key: ${{ secrets.CI_SERVICE_USER_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.NEXTFLOW_PROD_CI_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-          role-to-assume: arn:aws:iam::563295687221:role/NlpSandboxServiceAccount-ServiceRole-H1K0Q1UH2IX7
+          role-to-assume: arn:aws:iam::728882028485:role/workflows-nextflow-ci-service-account-ServiceRole-CTLXKJIOCRO4
```

Comment:
Is this really the correct ARN for the service account?

Reply:
@thomasyu888: This needs to be updated to the role ARN that was deployed for [this template](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/600-access/_tasks.yaml#L120-L131). You should be able to find this in the CloudFormation console under the `nlp-sandbox-ci-service-account` stack.